core-web/libs/sdk/client/src/lib/client/content/builders/collection/collection.ts

```diff
@@ -57,17 +63,30 @@ export class CollectionBuilder<T = unknown> {
         return this.#sortBy?.map((sort) => `${sort.field} ${sort.order}`).join(',');
     }
 
+    /**
+     * Returns the offset for pagination.
+     *
+     * @readonly
+     * @private
+     * @memberof CollectionBuilder
+     */
```

Comment:
Private methods are not expose to the consumer, is ok if we have docs, but just saying.